.eslintignore

```diff
@@ -8,4 +8,6 @@ docs
 packages/template/project/**
 app.playground.js
 type-test.ts
-packages/**/modular/dist/**
\ No newline at end of file
+packages/**/modular/dist/**
+packages/vertexai/__tests__/test-utils
+packages/vertexai/dist
```

Comment:
the .eslintignore file has a non-trivial conflict from #8262 - the file is completely gone as eslint config changed completely across major versions. You add new ignores to the new .eslint.mjs file following pattern there

note the glob / regex style changed just a bit but the pattern is easy to follow

https://github.com/invertase/react-native-firebase/blob/main/eslint.config.mjs#L17-L39

Reply:
I was the collision here and I had a moment, so I just handled the fresh merge from main and resolved the conflict 🚂